.github/workflows/go.yml

```diff
@@ -35,6 +35,9 @@ jobs:
   test:
     name: Test
     runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        artifactory: [ '7.9.2', '7.10.6', '7.11.2' ]
```

Comment:
Maybe the first one should be an older Artifactory (7.0.3)

Reply:
Let's try

Reply:
Earliest version on bintray is 7.2